### PR TITLE
XWIKI-20693: Livetable page size select doesn't have a name

### PR DESCRIPTION
**Pull request by Sereza7**
**Jira:** https://jira.xwiki.org/browse/XWIKI-20693

**PR Changes:**
* Added an aria-label to the livetable pagesizeselect Element

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/table/livetable.js [hunk 1]
```diff
@@ -1028,7 +1028,8 @@ var LiveTablePagination = Class.create({
    * @return an Element containing the select
    **/
   createPageSizeSelectControl: function() {
-    var select = new Element('select', {'class':'pagesizeselect'});
+    var select = new Element('select', {'class':'pagesizeselect',
+    'aria-label':"$escapetool.xml($services.localization.render('platform.livetable.selectPageSize.label'))"});
```
Comment (manuelleduc) [inline xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/table/livetable.js:1032]:
```suggestion
    'aria-label':"$escapetool.javascript($services.localization.render('platform.livetable.selectPageSize.label'))"});
```

File: xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/table/livetable.js [hunk 2]
```diff
@@ -1028,7 +1028,8 @@ var LiveTablePagination = Class.create({
    * @return an Element containing the select
    **/
   createPageSizeSelectControl: function() {
-    var select = new Element('select', {'class':'pagesizeselect'});
+    var select = new Element('select', {'class':'pagesizeselect',
+    'aria-label':"$escapetool.javascript($services.localization.render('platform.livetable.selectPageSize.label'))"});
```
Comment (manuelleduc) [inline xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/table/livetable.js:1032]: The formatting looks wrong around ':'. Also, isn't the line too long?

Comment (Sereza7) [reply xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/table/livetable.js]: Swapped the quotation marks :+1: 
L1040 was still longer. I fixed both to make it more readable.

File: xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/table/livetable.js [hunk 3]
```diff
@@ -1028,15 +1028,20 @@ var LiveTablePagination = Class.create({
    * @return an Element containing the select
    **/
   createPageSizeSelectControl: function() {
-    var select = new Element('select', {'class':'pagesizeselect'});
+    var select = new Element('select',
+      {'class':'pagesizeselect', 'aria-label':
+        '$escapetool.javascript($services.localization.render("platform.livetable.selectPageSize.label"))'
+      });
```
Comment (mflorea) [inline xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/table/livetable.js:1034]:
```suggestion
    var select = new Element('select', {
      'class': 'pagesizeselect',
      'aria-label': '$escapetool.javascript($services.localization.render("platform.livetable.selectPageSize.label"))'
    });
```

File: xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/table/livetable.js [hunk 4]
```diff
@@ -1028,15 +1028,20 @@ var LiveTablePagination = Class.create({
...
       var attrs = {'value':i, 'text':i};
       if (i == this.currentValue) {
         attrs.selected = true;
       } else {
         var prevStep = i - this.step;
         if (this.currentValue > prevStep && this.currentValue < i) {
-          select.appendChild(new Element('option', {'value':this.currentValue, 'text':this.currentValue, selected:true}).update(this.currentValue));
+          select.appendChild(new Element('option',
+            {'value':this.currentValue, 'text':this.currentValue, selected:true})
+          .update(this.currentValue));
```
Comment (mflorea) [inline xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/table/livetable.js:1044]:
```suggestion
          select.appendChild(new Element('option', {
            value: this.currentValue,
            text: this.currentValue,
            selected: true
          }).update(this.currentValue));
```

**Comment by Sereza7** [comment]
Should be ready to merge.

**Pull request merged**